doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+Welcome dear contributor, there are many ways to help this Badger. From quick
+and whimsical to involved and academic. From Javascript to Japanese. From
+algorithmic and O(n²) to grammatical and stylistic. There is surely a way you
+can help. Here's a quick orientation to guide your adventure:
+
+* Did we break a site? We want to fix it, so we wrote [this how-to
+  guide](/doc/fixing-broken-sites.md) for fixing broken sites.
+* Security vulnerabilities can be reported privately at
+  [vulnerabilities@eff.org](mailto:vulnerabilities@eff.org), the GPG key and
+  other information can be found [here](https://www.eff.org/security).
+* Translations are currently done on github. Here is an great
+  [example](https://github.com/EFForg/privacybadger/pull/1270) of a
+  contribution. See the [translations
+  label](https://github.com/EFForg/privacybadger/issues?utf8=%E2%9C%93&q=label%3Atranslations%20)
+  for related issues and more example pull requests.
+* Tests are crucial, so here's a guide on how to use them.
+<!---
+* Tell us about our bugs [here]()! Tips to make you're report 10x helpful are in this [bug tips document]().
+* Pull Requests welcome! Our PR workflow is documented [here]().
+* This short overview of Privacy Badger's architecture can help put things in perspective.
+* Developer setup and debugging
+* Documentation, brought you this document.
+-->
+
+
+Ignorance is a fleeting virtue. Sometimes our developers are blinded by
```

Comment:
Yeah, this needs to be cut down.

I want to encourage newcomers to report issues they have with onboarding. Often I think newcomers might might be self-conscious, and blame onboarding issues on themselves, rather than it actually being a problem with our documentation.

I think some degree of cutsieness is encouraging to people. It sets the tone for like "we won't yell at you if you ask a dumb question". But yeah, this